rusoto/credential/Cargo.toml

```diff
@@ -17,7 +17,7 @@ chrono = "0.4.0"
 dirs = "1.0.2"
 futures = "0.1.16"
 hyper = "0.12"
-hyper-tls = "0.2"
+#hyper-tls = "0.2"
```

Comment:
I think you could just go ahead and remove this line!

Reply:
🤦‍♂️